pymemcache/__init__.py

```diff
@@ -1,5 +1,3 @@
-__version__ = '2.1.1'
```

Comment:
It's possible some people expect this symbol (`pymemcache.__version__`) to be available. Removing it could break backwards compatibility.

Reply:
Great! Let's try that approach.

Reply:
Done.